integ-test/src/test/java/org/opensearch/sql/legacy/SQLFunctionsIT.java

```diff
@@ -61,70 +57,75 @@ protected void init() throws Exception {
 
   @Test
   public void functionFieldAliasAndGroupByAlias() throws Exception {
-    String query = "SELECT " +
-        "floor(substring(address,0,3)*20) as key," +
-        "sum(age) cvalue FROM " + TEST_INDEX_ACCOUNT + " where address is not null " +
-        "group by key order by cvalue desc limit 10  ";
+    String query =
+        "SELECT "
+            + "floor(substring(address,0,3)*20) as key,"
+            + "sum(age) cvalue FROM "
+            + TEST_INDEX_ACCOUNT
+            + " where address is not null "
+            + "group by key order by cvalue desc limit 10  ";
     final JSONObject result = executeQuery(query);
 
-
-    IntStream.rangeClosed(0, 9).forEach(i -> {
-          Assert.assertNotNull(result.query(String.format("/aggregations/key/buckets/%d/key", i)));
-          Assert.assertNotNull(
-              result.query(String.format("/aggregations/key/buckets/%d/cvalue/value", i)));
-        }
-    );
+    IntStream.rangeClosed(0, 9)
+        .forEach(
+            i -> {
+              Assert.assertNotNull(
+                  result.query(String.format("/aggregations/key/buckets/%d/key", i)));
+              Assert.assertNotNull(
+                  result.query(String.format("/aggregations/key/buckets/%d/cvalue/value", i)));
+            });
   }
 
   /**
    * todo fix the issue.
    *
-   * @see <a href="https://github.com/opendistro-for-elasticsearch/sql/issues/59">https://github.com/opendistro-for-elasticsearch/sql/issues/59</a>
+   * @see <a
+   *     href="https://github.com/opendistro-for-elasticsearch/sql/issues/59">https://github.com/opendistro-for-elasticsearch/sql/issues/59</a>
```

Comment:
a string longer than 100

Reply:
It'll be addressed by spotless in final `integ-test` spotless PR if it is a problem.